project/.travis/before_install_test.sh.twig

```diff
@@ -14,6 +14,10 @@ fi
 composer self-update --stable
 sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 
+{% if 'mongodb' in services %}
+pecl install --force mongodb-stable
+composer require "alcaeus/mongo-php-adapter=^1.0.0" --ignore-platform-reqs
```

Comment:
* This should be installed only on PHP7, should not be?
* You should add the `--no-update` option.